domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -116,6 +128,11 @@
 
   private static String          configPath               = null;
 
+  /**
+   * Used only if {@link #sharedPgPool} is true.
+   */
+  private static Map<Vertx,PgPool> pgPools = new HashMap<>();
```

Comment:
Could be final. (and so could connectionPool below)

Reply:
Thanks, fixed.